docs/guides/privileged_devices.md

```diff
@@ -0,0 +1,146 @@
+
+This should give you a good understanding of the kind of controls that are required and why they are necessary.
+
+If you have any questions then speak to your manager.
+
+#### Ensure that your device stays patched and up to date
+The organisation will push regular patches and updates to your device according to the standard patching schedule. You should normally apply these immediately.
+
+The only exception to this is where you need to maintain older versions of software to do compatibility testing. In this case, you should update as soon as possible.
+
+#### Ensure that you are using official online resources
+One of the main reasons for privileged access is to be able to make use of online resources.
+However, this freedom of access can also be a threat that may cause devices to be compromised or leak official information.
+
+For this reason, only officially approved online resources should be accessed from unmanaged devices.
+
+In reality, any resource that you access for legitimate work is likely to be "officially approved", but you are responsible for all of your online activity and will be accountable for any incidents resulting from inappropriate access.
+
+If you are not sure whether an online resource is considered to be officially approved then speak to your manager.
+
+#### Use public networks securely
+You will need to be more careful than usual whenever you are connecting to public networks as the privileged access you have can represent a higher information security risk should your communications be intercepted.
+In particular, the captive portals typically found in cafes and hotels that require a username and password may not provide secure end-to-end encryption.
+
+Whenever you are connecting to a public network, you must ensure that you first evaluate whether it is safe to do so.
+
+#### Backup and secure your data
+Privileged access allows us to install our own tooling. One consequence of this is that we inevitably store more data locally.
+
+You must ensure that any unique content you create is backed up to secure, corporate repositories such as OneDrive or SharePoint.
+
+You should also consider backing up some of your local settings and data, which could help if your device fails and needs to be fixed or repalaced.
+
+## Setup guidance
+The following sections will help you getting set up with some of the tools you are likely to use when you have privileged access.
+
+### Windows Subsystem for Linux
+If you are undertaking development, testing or prototyping activities, you are likely to benefit from being able to build and test in a Linux environment.
+
+The recommended approach to this if you have a Windows machine is to use the Windows Subsystem for Linux (WSL).
+
+You should ensure that you are using Windows 11 in order to get the best experience and install WSL2 rather than WSL1.
+
+Follow the below Microsoft guide to install the distro of your choice. The Ubuntu distro is recommended.
```

Comment:
Should we say why we recommend Ubuntu?  ie it has a lower learning curve, is commonly used, regularly patched, commonly used across Defra etc?

Reply:
Added some additional text 👍